src/frontend/running.rs

```diff
@@ -38,6 +44,14 @@ pub enum RunningInput {
     TogglePausePlotting,
 }
 
+#[derive(Debug)]
+pub enum CmdOut {
```

Comment:
Every other data structure here and in other files follows this pattern:
```suggestion
pub enum RunningCommandOutput {
```